src/Microsoft.Health.SqlServer/Identifier.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Health.SqlServer
+{
+    internal static class Identifier
```

Comment:
TODO: Leave comment about the source of this logic

Reply:
https://docs.microsoft.com/en-us/sql/relational-databases/databases/database-identifiers?view=sql-server-ver15